Kernel/Arch/x86/common/Interrupts.cpp

```diff
@@ -68,7 +68,7 @@ static EntropySource s_entropy_source_interrupts { EntropySource::Static::Interr
     }
 
 #define EH_ENTRY_NO_CODE(ec, title)                                            \
-    extern "C" void title##_asm_entry();	                                   \
+    extern "C" void title##_asm_entry();                                       \
     extern "C" void title##_handler(TrapFrame*) __attribute__((used));         \
     NAKED void title##_asm_entry(){                                            \
```

Comment:
A lot more snuck in, such as this brace not having a space before it.
Maybe take this out of the macro, reformat it (sans the asm), then put it back?

Reply:
if you don't feel like it, I could do that myself now :)

Reply:
Oof, hadn't even noticed that. Hang on.